deploy-edgecluster/render_edgeclusters.sh

```diff
@@ -252,15 +252,15 @@ EOF
     # Now process blocks for each master
     for master in $(echo $(seq 0 $(($(yq eval ".edgeclusters[${edgeclusternumber}].[]|keys" ${EDGECLUSTERS_FILE} | grep master | wc -l) - 1)))); do
         # Master loop
-        export CHANGE_EDGE_MASTER_PUB_INT=$(yq eval ".edgeclusters[${edgeclusternumber}].${cluster}.master${master}.nic_int_static" ${EDGECLUSTERS_FILE})
-        export CHANGE_EDGE_MASTER_MGMT_INT=$(yq eval ".edgeclusters[${edgeclusternumber}].${cluster}.master${master}.nic_ext_dhcp" ${EDGECLUSTERS_FILE})
+        export CHANGE_EDGE_MASTER_PUB_INT=$(yq eval ".edgeclusters[${edgeclusternumber}].[].master${master}.nic_int_static" ${EDGECLUSTERS_FILE})
```

Comment:
maybe i have it wrong but i cannot find the list '[]' in the yaml 
```console
$ yq e ".edgeclusters[0].[].master0" examples/config.yaml
Error: Bad expression, please check expression syntax
```
it looks like a dict so this syntx should not work
```shell
yq e ".edgeclusters[0].edgecluster1-name.master0" examples/config.yaml
ignore_ifaces: eno1 eno2
nic_ext_dhcp: eno4 # ext_dhcp -> DHCP
mac_ext_dhcp: "aa:ss:dd:ee:b0:10"
bmc_url: "<url bmc>"
bmc_user: "user-bmc" #TODO  may be clear test and render with script
bmc_pass: "user-pass" #TODO  may be clear test and render with script
root_disk: "/dev/sda"
storage_disk:
  - /dev/sdb
  - /dev/sdc
  - /dev/sde
  - /dev/sdd
  ````